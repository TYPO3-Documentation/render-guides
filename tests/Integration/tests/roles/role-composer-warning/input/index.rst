==============
Document Title
==============

You can also install extensions like :composer:`news` and
:composer:`georgringer/news-administration`.
