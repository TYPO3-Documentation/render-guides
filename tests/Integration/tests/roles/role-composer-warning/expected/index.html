<!-- content start -->
                <section class="section" id="document-title">
            <h1>Document Title<a class="headerlink" href="#document-title" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline"><i class="fa-solid fa-paragraph"></i></a></h1>

    <p>You can also install extensions like news and
georgringer/news-administration.</p>

    </section>
        <!-- content end -->
